docker/Dockerfile

```diff
@@ -41,10 +38,18 @@ RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --defau
 ENV PATH="/root/.cargo/bin:$PATH" \
     PGX_HOME=/usr/lib/postgresql/${PG_VERSION_MAJOR}
 
+# Copy Cargo.toml because we need to extract the pgrx version
+WORKDIR /tmp
+COPY pg_search/Cargo.toml ./
+
 # Intall the pgrx version of pg_search and pg_analytics
-RUN cargo install --locked cargo-pgrx --version "${PGRX_VERSION}" && \
+RUN PGRX_VERSION=$(sed -n 's/^pgrx *= *"=*\([0-9.]*\)"/\1/p' Cargo.toml) && \
+    echo "PGRX_VERSION=$PGRX_VERSION" && \
+    cargo install --locked cargo-pgrx --version "${PGRX_VERSION}" && \
```

Comment:
Keep the regular expression here, as we cannot use 'cargo tree' in this instance.

Reply:
Yeah, I was thinking about that.  I think it's fine.  The only option I came up with was to copy the entire source tree into the docker image, do the cargo tree stuff and then delete the source tree.

That seems like a lot of extra effort for not much gain.  But if in the future this regex starts to give us problems, we can take a look at a different approach.